### PR TITLE
Style

### PR DESCRIPTION
**Pull request by Mahdi-Niazi**
### In this milestone I:

- Add style to navbar
- Add style to book lists
- Add style to input form
- Add style to category page

### CODE REVIEW

File: src/components/styles/form.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+.container form button {
+  width: 25%;
+  background-color: #0290ff;
+  color: white;
+  font-size: 18px;
+  font-weight: bold;
+  border: none;
+  cursor: pointer;
+}
```
Comment (binhussen) [inline src/components/styles/form.css]: - Great Job for having this Good UI 👍 I think for this milestone you have to stick to the given Design and update font size, position, and space to fit the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md) 


#### Given Design 👇


#### Your Design  👇

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Mahdi-Niazi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Good UI✔️
- Well Structured Code ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/styles/form.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+.container h2 {
+  margin: 1rem 0.2rem;
+  color: gray;
+}
+
+.container form {
+  padding: 1rem 0.3rem;
+  display: flex;
+  gap: 3rem;
+}
```
Comment (ridwanediallo) [inline src/components/styles/form.css:17]: - [ ] You did a fantastic job implementing the improvements that the previous reviewer had requested 👏. However, I believe you overlooked one modification, which is to make sure the form section is correctly oriented to the left and right with the cards as in the original design. Please ensure to fix that so your design resembles the template as closely as possible. 👌 😊

#### This is what the reviewer pointed out :point_down: 
Screenshot from 2023-08-26 10-52-59

#### This is what we still have :point_down: 
Screenshot from 2023-08-26 10-57-34

### This is how the template looks like :point_down: 
Screenshot from 2023-08-26 11-07-29

So this :point_down:  is a suggestion to help you fix the issue. 😊

```suggestion
.container {
  margin: 2rem 6.25rem;
  border-radius: 10px;
  margin-bottom: 10rem;
}

.container h2 {
  margin: 1rem 0;
  color: gray;
}

.container form {
  padding: 1rem 0;
  display: flex;
  gap: 3rem;
}
```

What I did here is I removed:
- the `width: 86%;` to the `container` and updated the **margin-left and right** from `auto` to `6.5rem`. 
- updated the **margin-left and right** from `0.2rem` to `0` added to the `h2`
- updated the **margin-left and right** from `0.3rem` to `0` added to the `form`

If you follow their recommendations, the outcome will be as follows :point_down:   which resembles the original design in my opinion.😊

#### Result :point_down: 
Screenshot from 2023-08-26 11-22-28

**Review by ridwanediallo — APPROVED** [review]
# Changes Required ♻️
## Hi @Mahdi-Niazi, 


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Awesome implementing most of the changes requested by the previous reviewer. ✔ 💪 💪 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**